javascripts/govuk/multivariate-test.js

```diff
@@ -71,18 +71,11 @@
   };
 
   MultivariateTest.prototype.setCustomVar = function(cohort) {
-    window._gaq = window._gaq || [];
-    window._gaq.push([
-      '_setCustomVar',
-      this.customVarIndex,
-      this.cookieName(),
-      cohort,
-      2 // session level
-    ]);
+    GOVUK.analytics.setDimension(this.customVarIndex, this.cookieName(), cohort, 2) // session level
```

Comment:
Can this be split out on new lines like it was previously so that it is obvious that the `2` is the part that the `// session level` comment relates to.